src/LearnositySdk/Utils/Uuid.php

```diff
@@ -2,11 +2,13 @@
 
 namespace LearnositySdk\Utils;
 
+require_once __DIR__ . '/../../../vendor/paragonie/random_compat/lib/random.php';
+
 // Thanks to Andrew Moore http://www.php.net/manual/en/function.uniqid.php#94959
 
 class Uuid
 {
-    public static function generate($type = 'v4', $namespace = null, $name = null)
+    public static function generate($type = 'uuidv4', $namespace = null, $name = null)
```

Comment:
The default type `uuidv4` is not handled in the switch below. Do we actually want to change that default? The new lib should be a transparent replacement, shouldn't it?

This default type would go into the `default` case, which then uses the old `v4` method.

I think we might as well move the v4 method out of the way (or rename it to, say `legacy-v4`?), and replace it altogether with your new uuidv4.